src/components/App/App.tsx

```diff
@@ -10,9 +10,38 @@ import WeddingPhotoList from '../WeddingPhotoList/WeddingPhotoList'
 import LandingPage from '../LandingPage/LandingPage'
 import GuestListForm from '../GuestList/GuestList'
 import AddWeddingForm from '../AddWeddingForm/AddWeddingForm'
+import { getWeddings } from '../../apiCalls'
 
+type Wedding = {
+  id: number;
+  name: string;
+  email: string;
+  date: string;
+  image: string;
+}
 
 const App = () => {
+  const [weddings, setWeddings] = useState<Wedding[]>([])
+  const [errorMessage, setErrorMessage] = useState('')
+  const [hasError, setError] = useState(false)
+
+  useEffect(() => {
+    const allWeddings = async () => {
+      const result = await getWeddings()
+      if(result.length > 0) {
+        result.forEach((wed: any) => {
+          wed.date = new Date(wed.date)
+        })
+        let sortedResult = result.sort((a: any, b: any) => a.date - b.date)
+        setWeddings(sortedResult)
+      } else {
+        setError(true)
+        setErrorMessage(result)
+      }
+    }
+    allWeddings()
+  }, [])
+
```

Comment:
I wonder how we get around data showing up empty with the useEffect on the first render.